src/nodejs.nim

```diff
@@ -64,4 +66,6 @@ export
   jsdiagnostics_channel,
   jswasi,
   jswebsockets,
-  jscluster
+  jscluster,
+  jsmultisync,
+  jsxmlhttprequest
```

Comment:
I was thinking of just removing this file, because people should import modules individually,
instead of just importing everything all at once, it is like too much, 
it was created originally for Documentation purposes anyway, what do you think ?.